### ISSUE TITLE
Listing files owned/uploaded by Account Manager/Administrator

### ISSUE THREAD

**Issue by olimax** [ignacionelson/ProjectSend#434]
Am I just missing something here?
I can't see a way to list files uploaded by a specific Account manager/Admin 
Nor to be able to assign files to them

**Comment by ignacionelson** [comment]
No, you are right. Filtering by uploader is not available. I'll work on this and add it as a filter on the manage files page today.
On the other hand, assigning files to users (instead of clients) is not possible, as assigning means giving a client view/download permissions on the file. Admins and account managers can see all files already.

**Comment by olimax** [comment]
That makes sense. 
Much appreciated.
Many thanks.

**Comment by ignacionelson** [comment]
Thank your for your suggestions! Can't believe I missed something like this

**Comment by ignacionelson** [comment]
This issue is being closed as part of a bulk cleanup. ProjectSend has undergone significant changes since this was reported (including r1720 and r1994), and many older issues no longer apply to the current codebase.

If you are still experiencing this problem on the latest version, please open a new issue with updated details and we'll be happy to look into it. Thank you for your contribution!
